run_federation_tests.py

```diff
@@ -151,9 +154,28 @@
         raise
 
     finally:
-        logging.warning('collecting logs [{}]'.format(output_directory))
-        logs.collect_logs(ctx.docker_client, ctx.irods_containers(), output_directory)
+        if args.save_logs:
+            try:
+                logging.error('collecting logs [{}]'.format(output_directory))
+
+                # collect the usual logs
+                logs.collect_logs(ctx.docker_client, ctx.irods_containers(), output_directory)
+
+                # TODO: This produced errors... why do the federation tests show up somewhere else?
```

Comment:
where do they appear?

Reply:
The logs are only appearing in the container that actually ran the test. Adjusted.